bundles/org.openhab.binding.bloomsky/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,547 @@
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+
+	<!-- BloomSky STORM Personal Weather Station Thing -->
+	<thing-type id="storm">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge-api"/>
+		</supported-bridge-type-refs>
+		<label>BloomSky Storm Personal Weather Station</label>
+		<description>Provides weather data from the BloomSky personal weather stations account API</description>
+
+		<channel-groups>
+			<!-- Group for Storm weather observations -->
+			<channel-group id="storm-observations" typeId="storm-observations">
+				<label>Storm Weather Observations</label>
+				<description>This is the Storm Weather Station observations</description>
+			</channel-group>
+		</channel-groups>
+		<representation-property>stormAssociatedWithSkyDeviceID</representation-property>
+	</thing-type>
+
+	<channel-group-type id="storm-observations">
+		<label>Storm Device Weather Observations</label>
+		<description>This is the weather sensor observations from the Storm device</description>
+		<channels>
+			<channel typeId="stormUVIndex" id="stormUVIndex"></channel>
+			<channel typeId="system.wind-direction" id="windDirectionAngle"></channel>
+			<channel typeId="windDirectionCompass" id="windDirectionCompass"></channel>
+			<channel typeId="rainDaily" id="rainDaily"></channel>
+			<channel typeId="windGust" id="windGust"></channel>
+			<channel typeId="windChill" id="windChill"></channel>
+			<channel typeId="sustainedWindSpeed" id="sustainedWindSpeed"></channel>
+			<channel typeId="rainRate" id="rainRate"></channel>
+			<channel typeId="rain24h" id="rain24h"></channel>
+			<channel typeId="stormTimeStamp" id="stormTimeStamp"></channel>
+		</channels>
+	</channel-group-type>
+
+	<!-- Storm Specific Channel Types -->
+
+	<channel-type id="stormUVIndex">
+		<item-type>Number</item-type>
+		<label>UV Index</label>
+		<description>UV index</description>
+		<state readOnly="true" pattern="%d"></state>
+	</channel-type>
+
+	<channel-type id="rainRate">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation Rate</label>
+		<description>Precipitation rate</description>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%/hr"/>
+	</channel-type>
+
+	<channel-type id="sustainedWindSpeed">
+		<item-type>Number:Speed</item-type>
+		<label>Sustained Wind Speed</label>
+		<description>Sustained Wind Speed</description>
+		<category>Wind</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rain24h">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation 24h</label>
+		<description>Precipitation last 24 hours</description>
+		<category>Rain</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rainDaily">
+		<item-type>Number:Length</item-type>
+		<label>Precipitation Total Past 24h</label>
```

Comment:
We prefer the label to be 2-3 words max. Suggest:
Todays Rain
Rain Today
Rain Daily

I tend to prefer keeping it the same as the channel ID as it makes it easier for the textual people. The label is used to auto create the name for the items when you create equipment from a thing.

Reply:
Went through entire list of labels and made changes as suggested.